src/autoscaler/api/brokerserver/broker_handler.go

```diff
@@ -104,6 +128,166 @@ func (h *BrokerHandler) CreateServiceInstance(w http.ResponseWriter, r *http.Req
 	}
 }
 
+func (h *BrokerHandler) UpdateServiceInstance(w http.ResponseWriter, r *http.Request, vars map[string]string) {
+	instanceId := vars["instanceId"]
+
+	body := &models.InstanceUpdateRequestBody{}
+	bodyBytes, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		h.logger.Error("failed to read service update request body", err, lager.Data{"instanceId": instanceId})
+		writeErrorResponse(w, http.StatusInternalServerError, "Failed to read request body")
+		return
+	}
+	err = json.Unmarshal(bodyBytes, body)
+	if err != nil {
+		h.logger.Error("failed to unmarshal service update body", err, lager.Data{"instanceId": instanceId, "body": string(bodyBytes)})
+		writeErrorResponse(w, http.StatusBadRequest, "Invalid request body format")
+		return
+	}
+
+	if instanceId == "" || body.ServiceID == "" {
+		h.logger.Error("failed to update service instance when trying to get mandatory data", nil, lager.Data{"instanceId": instanceId, "serviceId": body.ServiceID, "planId": body.PlanID})
+		writeErrorResponse(w, http.StatusBadRequest, "Malformed or missing mandatory data")
+		return
+	}
+
+	if body.Parameters == nil || body.Parameters.DefaultPolicy == nil {
+		h.logger.Error("failed to update instance, only default policy updates allowed", nil, lager.Data{"instanceId": instanceId, "serviceId": body.ServiceID, "planId": body.PlanID})
+		writeErrorResponse(w, http.StatusUnprocessableEntity, "Failed to update service instance: Only default_policy updates allowed")
+		return
+	}
+	updatedDefaultPolicy := string(*body.Parameters.DefaultPolicy)
+
+	if emptyJSONObject.MatchString(updatedDefaultPolicy) {
+		// accept an empty json object "{}" as a default policy update to specify the removal of the default policy
+		updatedDefaultPolicy = ""
+	}
```

Comment:
So we must remove the default policy by updating it to  "{}", right?   If no parameter is passing, could we treat it as the removal as well?   I am not sure whether we need to document this constraint in document.

Reply:
We could do this also, but we would need to distinguish between "no parameters passed" and no "default_policy passed".

> Note: The parameters specified are expected to be the values specified by an end-user. Whether the user chooses to include the complete set of configuration options or just a subset (or even none) is their choice. How a Service Broker interprets these parameters (including the absence of any particular parameter) is out of scope of this specification - with the exception that if this field is not present in the request then the Service Broker MUST NOT change the parameters of the instance as a result of this request.

https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#body-6

Reply:
As we discussed to remove the "default_policy" layer to make the policy definition format consistent,  here we only need to discuss the exact cmd to trigger the policy removal. 
To make it more clearly,   I listed the possible cmds as below: 
`cf update-service <instance-name>`   --> Expected result:  Remove?  or treat it as "no changes" ? 
`cf update-service <instance-name>  -c ""`  --> Expected result:  Invalid parameter? 
`cf update-service <instance-name>  -c "{}" ` --> Expected result:  remove policy